### PR TITLE
Added the social media links

### PR DESCRIPTION
**Pull request by SirAbhi13**

### CODE REVIEW

File: dev/index.html [hunk 1]
```diff
@@ -935,10 +935,18 @@ <h3><a href="#">Lorem Ipsum </a></h3>
...
-        <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
-        <a href="#"><i class="fa-brands fa-discord"></i></a>
-        <a href="#"><i class="fas fa-envelope"></i></a>
+        <a href="https://www.instagram.com/nexus_whatthehack/"
+          ><i class="fa-brands fa-instagram"></i
+        ></a>
+        <a href="https://www.linkedin.com/company/nexus-what-the-hack/"
+          ><i class="fa-brands fa-linkedin-in"></i
+        ></a>
+        <a href="https://discord.gg/uskRaeshyW"
```
Comment (pulkit-30) [inline dev/index.html:944]: add https://discord.gg/R53QVZJ2fn link for discord

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

File: dev/index.html (hunk 1 above)
Comment (SirAbhi13) [reply dev/index.html:944]: The link I added is permanent and no limit invitation link.

I don't think it would matter.

Comment (SirAbhi13) [reply dev/index.html:944]: Ah nvm, it's the elixir link.

My bad. 😅

Comment (pulkit-30) [reply dev/index.html:944]: @dewank07

Comment (pulkit-30) [reply dev/index.html:944]: > Ah nvm, it's the elixir link. 
> My bad. 😅

Np

**Review by pulkit-30 — APPROVED** [review]

**Pull request merged**